js/app.js

```diff
@@ -79,5 +84,48 @@ const renderUniqueKeywords = (arrayOfImages) => {
 
 $(document).ready(function () {
   Animal.getAnimalsFromFile();
+  $('#page2').on('click', changeToPage);
 })
 
+const changeToPage = () => {
+  console.log('we made it!')
+  const filePath2 = './data/page-2.json';
+  const filePath1= './data/page-1.json';
+  const fileType = 'json';
+  const buttonValue = $('button').val();
+  $('#photo-template').siblings().remove();
+  $('#keyword-dropdown').children().remove();
+  allAnimals = [];
+  keyWords = [];
+  if (buttonValue === 'page2') {
+    $('#page2').val('page1').text('Page One');
```

Comment:
i like this part, where you just have one button that changes depending on what is showing, rather than having two separate buttons